.github/workflows/testing.yml

```diff
@@ -62,12 +62,12 @@ jobs:
           set -vxeo pipefail
           conda env list
           mamba install -c conda-forge -y numpy matplotlib
-          pip install -v pypylon
+          pip install -r requirements.txt
+          pip install -r requirements-dev.txt
```

Comment:
After that line, please also install the package itself:
```bash
pip install -v .
```
In this case, the dependencies from `requirements.txt` will be installed automatically, and line 65 can be removed.